src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -172,6 +172,15 @@ class RemoteMachineTrainingService implements TrainingService {
         }
     }
 
+    /**
+     * Get trial job log
+     * @param _trialJobId ID of trial job
+     * @param _logType 'TRIAL_LOG' | 'TRIAL_STDERR'
+     */
+    public async getTrialLog(_trialJobId: string, _logType: LogType): Promise<string> {
```

Comment:
The doc string should be written in base class.

Reply:
Agreed. I write doc string in this class because I found other functions like `getTrialJob` also have a doc string in this file. Should I remove them all?